app/Mail/ContactMail.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class ContactMail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * @var string
+     */
+    public $name;
+    /**
+     * @var string
+     */
+    public $email;
+    /**
+     * @var string
+     */
+    public $message;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(string $subject, string $message, ?string $name, string $email)
+    {
+        $this->subject = $subject;
```

Comment:
there is no subject public var

Reply:
We don't need to declare it since it's already declared on the parent Mailable class. It's enough to just set it and it will automatically be handled by the Mailable class to put it as the email's subject.